_drafts/2023-03-01-announcing-call-for-proposals.md

```diff
@@ -11,15 +11,15 @@ layout: post
 title: The Call for Proposals for DjangoCon US 2023 Is Now Open!
 ---
 
-The DjangoCon 2023 organizers are excited that DjangoCon US 2023 is now open for talk submissions: [call for proposals](https://pretalx.com/djangocon-2023/cfp)! The deadline for submissions is June 10th, 2023 [AoE](https://time.is/compare/0000_10_June_2023_in_Anywhere_on_Earth). As long as it’s still June 10th anywhere on earth, you can submit your proposal.
+The DjangoCon 2023 organizers are excited that DjangoCon US 2023 is now open for talk submissions: [call for proposals](https://pretalx.com/djangocon-2023/cfp)! The deadline for submissions is May 15th, 2023 [AoE](https://time.is/compare/0000_15_May_2023_in_Anywhere_on_Earth). As long as it’s still May 15th anywhere on earth, you can submit your proposal.
```

Comment:
"The DjangoCon 2023 organizers are excited that DjangoCon US 2023" is a lot of "DjangoCon 2023" for one sentence. For announcements, I would avoid the third-person because it sounds cold and just saw, "We are excited..."

Reply:
Also we're dropping AoE this year in favor of 6 PM EDT. That was the source of far too much confusion last year. https://github.com/djangocon/2023.djangocon.us/pull/28/files#diff-2ee5892bdf01113ac07c8e4e5b108b456d66e2ebc799b880f0edd3d9de505141R23